package-lock.json

```diff
@@ -1,9868 +1,8 @@
 {
   "name": "elm-chorus",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
You might want to use a newer version of npm (version 14 at least) as we would like to use the newer version of the package lock